docs/api/v2.yaml

```diff
@@ -83,6 +83,37 @@ paths:
           $ref: '#/components/responses/NotFoundError'
         '500':
           $ref: '#/components/responses/ServerError'
+  /reconciliation-report/{date}:
+    get:
+      summary: fetches a list of docket entries served on a date
+      operationId: 
```

Comment:
Just flagging so it doesn't get lost, but I this needs a string value to pass validation. I'm pretty sure @kkoskelin has a fix incoming.

Reply:
in the queue: https://github.com/flexion/ef-cms/pull/8067